CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Changes
 - Added `SignatureActivity` to accept and save user signature
 - Increase error text sizes
+- Added `TeleConsultSuccessScreen` to notify teleconsultation recorded successfully
```

Comment:
This could be an "Internal" change since the screen itself is not accessible at this point.